tests/unit/mathematics/linalg/operations/Eigen3_operations_unittest.cc

```diff
@@ -465,6 +465,63 @@ TEST(LinalgBackendEigen, SGMatrix_cross_entropy)
 	EXPECT_NEAR(ref, result, 1e-15);
 }
 
+TEST(LinalgBackendEigen, SGMatrix_pinvh)
+{
+	float64_t A_data[] = {2.0, -1.0, 0.0,
+												-1.0, 2.0, -1.0,
+													0.0, -1.0, 2.0};
+								
+	float64_t result_data[] = {0.75, 0.5, 0.25,
+															0.5, 1.0, 0.5,
+															0.25, 0.5, 0.75};
+																		
+	SGMatrix<float64_t>A(A_data, 3, 3, false);
+	SGMatrix<float64_t>result(result_data, 3, 3, false);
+	
+	SGMatrix<float64_t> I(3, 3);
```

Comment:
is there no API for 
`auto I = linalg::identity(3);`?

Reply:
no i did not find any

Reply:
i guess data type issues, ok leave it as it it